rides/templates/extend/base.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
wait, was there not a base template before?

Reply:
base.html was taken from ComputerScienceHouse/packet